Volha_Ivanova/Module_1_Java/Java_Fundamentals/HomeWork1/src/com/epam/cdp/volha_ivanova/task1/Runner.java

```diff
@@ -0,0 +1,45 @@
+package com.epam.cdp.volha_ivanova.task1;
+
+public class Runner {
+	/**
+	 * @author Volha_Ivanova 
+	 * Input : array of integers from -10 to 10; 
+	 * Output: Sum of elements that have event index
+	 * @param min Min value of array of integers
+	 * @param max Max value of array of integers
+	 */
+
+	private static int[] array = new int[20];
+
+	public static void main(String[] args) {
+
```

Comment:
generally all files are OK. 
Just one minor note. Your formatting looks a little bit strange - a lot of tabs. I believe auto formatting shouldn't do that, but maybe I make mistake. Also be aware about useless empty strings. 
All the format-notes make your code difficult for read and difficult for review.

You can change it before tomorrow. Tomorrow I will evaluate your code finally.